tests/unit/lms/product/canvas/_plugin/misc_test.py

```diff
@@ -156,6 +156,23 @@ def test_get_deeplinking_launch_url(self, plugin, pyramid_request):
             == "http://example.com/lti_launches?param=value"
         )
 
+    @pytest.mark.parametrize("url_param", (None, sentinel.from_url))
+    @pytest.mark.parametrize("custom_param", (None, sentinel.from_custom))
+    def test_get_deep_linked_assignment_configuration(
```

Comment:
There was not an explicit test for this method. It seems it was tested via other methods that use it.

Added one focused on this  change.